cdap-standalone/bin/cdap.sh

```diff
@@ -109,7 +109,7 @@ fi
 NODE_INSTALL_STATUS=$(program_is_installed node)
 if [ "x$NODE_INSTALL_STATUS" == "x1" ]; then
   die "Node.js is not installed
-Please install Node.js - we recommend any version of Node.js greater than v0.10.0."
+Please install Node.js - we recommend any version of Node.js from v0.10.36 through v0.12.*"
```

Comment:
I mean, the current message seems to imply that we dont support Node 4.x.x or 5.x.x. Right?